deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,40 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+body{
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+div{
+    display: flex;
+    flex-direction: column;
+}
+
+main{
+    display: flex;
+    flex-direction: column;
+    flex-basis: auto;
+    min-width: 80vw;
+    flex-grow: 1;
+    flex-shrink: 1;
+}
+
+nav{
```

Comment:
Zelfde geldt hier voor de nav. Mag in principe nooit een class ongebruikt in html of CSS laten staan.

Reply:
Oh top dank je voor de info.